ovs-p4rt/sidecar/client/ovsp4rt_client.h

```diff
@@ -24,9 +24,6 @@ class Client : public ClientInterface {
   // Connects to the P4Runtime server.
   virtual absl::Status connect(const char* grpc_addr);
 
-  // Returns a pointer to the ovsp4rt session object.
-  virtual OvsP4rtSession* session() const { return session_.get(); }
```

Comment:
Isnt this needed for future use?

Reply:
I thought so at first, but I've changed my mind.

The purpose of the `Client` class is to replace the discrete logic that interacts with the P4Runtime server with an object that implements an abstract interface.

- `ClientInterface` is an abstract class that defines the interface.
- `Client` is an implementation of `ClientInterface` that replaces the current discrete logic.
- `ClientMock` is a mock implementation of `ClientInterface`.

See issue https://github.com/ipdk-io/networking-recipe/issues/701 for more information.

Reply:
Ack